controller/server_status.py

```diff
@@ -170,7 +183,7 @@ class PodCondition:
     @classmethod
     def from_dict(cls, condition_dict: Dict[str, str]):
         return cls(
-            type=PodConditionsEnum(condition_dict["type"]),
+            type=PodConditionsEnum.from_string(condition_dict["type"]),
```

Comment:
With the above suggestion, this code could be reverted to its original state.

Reply:
Ok cool I did not know such a thing existed for enums.

Reply:
Done. That is pretty neat. Thanks for pointing that out.